ArduPlane/pullup.cpp

```diff
@@ -115,7 +115,7 @@ bool GliderPullup::verify_pullup(void)
     switch (stage) {
     case Stage::WAIT_AIRSPEED: {
         float aspeed;
-        if (ahrs.airspeed_estimate(aspeed) && aspeed > airspeed_start) {
+        if (ahrs.airspeed_estimate(aspeed) && (aspeed > airspeed_start || ahrs.pitch_sensor*0.01 > pitch_start)) {
```

Comment:
What will happen in the case of a release, being followed by a flat fall?
The airspeed will be low, but the pitch will pass this new test.

Will this mean that the stage will transition to WAIT_PITCH, which will then go into WAIT_LEVEL and return true with a low airspeed (much lower than expected)?

Reply:
indeed, but this is much better than staying in WAIT_AIRSPEED forever
note that glider pullup is designed to be used with the glider being held by its tail